### PR TITLE
Update FLUTTER_VERSION in cirrus config

### LINKED ISSUE #51: New latest version 1.12.13+hotfix.9
Hi, could you please add the `1.12.13+hotfix.9` image to docker hub? 

Thanks

### PR DESCRIPTION
**Pull request by neokree**
Updates 
- latest
- stable
- beta
- dev

branches.

This fixes #51

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -7,12 +7,12 @@ flutter_docker_builder:
...
     matrix:
     - DOCKER_TAG: latest
-      FLUTTER_VERSION: v1.12.13+hotfix.8
+      FLUTTER_VERSION: v1.12.13+hotfix.9
     - DOCKER_TAG: stable
-      FLUTTER_VERSION: v1.12.13+hotfix.8
+      FLUTTER_VERSION: v1.12.13+hotfix.9
     - DOCKER_TAG: beta
-      FLUTTER_VERSION: v1.15.17
+      FLUTTER_VERSION: v1.17.0-dev.3.1
```
Comment (fkorotkov) [inline .cirrus.yml:14]: It seems Flutter team dropped `v` from the tags from 1.17 version.

```suggestion
      FLUTTER_VERSION: 1.17.0-dev.3.1
```

File: .cirrus.yml [hunk 2]
```diff
@@ -7,12 +7,12 @@ flutter_docker_builder:
...
     - DOCKER_TAG: dev
-      FLUTTER_VERSION: v1.16.1
+      FLUTTER_VERSION: v1.18.0-dev.3.0
```
Comment (fkorotkov) [inline .cirrus.yml:16]:
```suggestion
      FLUTTER_VERSION: 1.18.0-dev.3.0
```

File: .cirrus.yml (hunk 1 above)
Comment (neokree) [reply .cirrus.yml]: Yep you're right, I committed a new version removing the v from beta and dev

**Comment by fkorotkov** [comment]
Thank you, @neokree! 🙌

**Pull request merged**